tests/VolumeSlider-test.js

```diff
@@ -33,6 +33,34 @@ describe('<VolumeSlider />', () => {
     expect(slider.find('.VolumeSlider-intent').props().style.height).to.equal('91%')
   })
 
+  it('renders with correct sizing when vertical', () => {
+    const slider = shallow(<VolumeSlider direction={ControlDirection.VERTICAL} isEnabled={true} volume={0.6} />)
+
+    slider.setState({ currentIntent: 0.2 })
+
+    const value = slider.find('.VolumeSlider-value')
+    const intent = slider.find('.VolumeSlider-intent')
+    const handle = slider.find('.VolumeSlider-handle')
+
+    expect(value.props().style).to.eql({ height: '60%' })
+    expect(intent.props().style).to.eql({ height: '20%' })
+    expect(handle.props().style).to.eql({ bottom: '60%' })
+  })
+
+  it('renders with correct sizing when horizontal', () => {
+    const slider = shallow(<VolumeSlider direction={ControlDirection.HORIZONTAL} isEnabled={true} volume={0.6} />)
+
+    slider.setState({ currentIntent: 0.2 })
+
+    const value = slider.find('.VolumeSlider-value')
+    const intent = slider.find('.VolumeSlider-intent')
+    const handle = slider.find('.VolumeSlider-handle')
+
+    expect(value.props().style).to.eql({ width: '60%' })
+    expect(intent.props().style).to.eql({ width: '20%' })
+    expect(handle.props().style).to.eql({ left: '60%' })
+  })
+
```

Comment:
Can we also add a test that the `direction` prop is passed on to `RangeControlOverlay`?